src/cascadia/TerminalSettingsEditor/Microsoft.Terminal.Settings.Editor.vcxproj.filters

```diff
@@ -22,6 +22,7 @@
       <Filter>Converters</Filter>
     </Midl>
     <Midl Include="SettingContainer.idl" />
+    <Midl Include="GlobalSettingsViewModel.idl" />

```

Comment:
(This would be a separate PR) Would there be any value to introducing these filters:
- view models
- views

And pulling out ProfileViewModel, KeyBindingViewModel, etc. into their own files and throwing them in the filter? Those classes are big enough that they could be their own objects tbh.